README.md

```
@@ -1,16 +1,186 @@
+├── xrpl/              # XRPL 클라이언트 서비스
+├── embeddings/        # 벡터 임베딩 생성
+└── outbox/            # Outbox 패턴 (Change Stream 기반)
+```
+
+## 아키텍처
+
+graph LR
+    %% Client Zone
+    Client((Client\nBuyer/Seller))
+    
+    %% Application Zone
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)